scripts/demo_overlay.py

```diff
@@ -183,7 +183,7 @@ def _DoAcquisition(future, escan, ccd, detector, light):
         detector.data.unsubscribe(_ssOnSEMImage)
 
     return optical_image_1, optical_image_2, optical_image_3,
-    detector.data._electron_image
+    detector.data._electron_image  # TODO code unreachable??
```

Comment:
Why would it be unreachable?
Admittedly, it could be on the same line as the return, to make it easier to read.